### PR TITLE
Added Suicide Squad 2016 for HacktoberFest

### PR DESCRIPTION
**Pull request by dkarakay**

### CODE REVIEW

File: movies/2016/suicide-squad.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Batman v Superman: Dawn of Justice",
```
Comment (haroon-sheikh) [inline movies/2016/suicide-squad.json]: can you correct this please

File: movies/1982/blade-runner.json [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (haroon-sheikh) [inline movies/1982/blade-runner.json]: can you remove this

File: movies/2016/suicide-squad.json (hunk 1 above)
Comment (dkarakay) [reply movies/2016/suicide-squad.json]: Ok

File: movies/1982/blade-runner.json (hunk 2 above)
Comment (dkarakay) [reply movies/1982/blade-runner.json]: I did it

**Comment by dkarakay** [comment]
#68

**Pull request closed without merging**